.github/workflows/validate.yml

```
@@ -45,7 +45,7 @@ jobs:
           cache: false
 
       - name: Set up Python
-        uses: actions/setup-python@0b93645e9fea7318ecaed2b359559ac225c90a2b # v5.3.0
+        uses: actions/setup-python@42375524e23c412d93fb67b49958b491fce71c38 # v5.4.0
         with:
           python-version: "3.8"
```

We're 3.9 in the generator now as well. 👍 

```suggestion
          python-version: "3.9"
```